dissect/target/helpers/loaderutil.py

```diff
@@ -38,12 +41,17 @@ def add_virtual_ntfs_filesystem(
         fs.ntfs = ntfs.ntfs
 
 
-def _try_open(fs, path):
+def _try_open(fs, path) -> None:
```

Comment:
```suggestion
def _try_open(fs: Filesystem, path: str) -> None:
```
And add the import for `from dissect.target.filesystem import Filesystem`